torch_xla/csrc/ops/sum.cpp

```diff
@@ -44,14 +44,15 @@ xla::Shape NodeOutputShape(const Value& input,
 
 }  // namespace
 
-Sum::Sum(const Value& input, std::vector<xla::int64> dimensions,
```

Comment:
Leave std::vector and move.
We have had this discussion before :)

Reply:
I know, I had to change it for a different reason. Let me try to change it back to see what was the problem.

Reply:
Hmm, turns out I could change it back.